de_hvbg/de_hvbg_README.txt

```diff
@@ -0,0 +1,16 @@
+# de_hvbg_README.txt
+
+The files in this section result from the conversion of datasets originating from [Hessische Verwaltung für Bodenmanagement und Geoinformation](https://www.hvbg.hessen.de)
+
+## Included grids
+
+### Germany (Hessen): DE_DHDN -> ETRS89
+
+*Source*: [HVBG Downloadcenter](https://www.gds.hessen.de/INTERSHOP/web/WFS/HLBG-Geodaten-Site/de_DE/-/EUR/ViewData-Start/1394640044?JumpTarget=ViewRequisitionProcess-Restart&IsOnlineBasket=false&ProductUUID=B9MKFwNVKxMAAAF.AeJVXZuMhttps://www.gds.hessen.de/INTERSHOP/web/WFS/HLBG-Geodaten-Site/de_DE/-/EUR/ViewData-Start/1394640044?JumpTarget=ViewRequisitionProcess-Restart&IsOnlineBasket=false&ProductUUID=B9MKFwNVKxMAAAF.AeJVXZuM)
+
+*Format*: GeoTIFF converted from NTv2                                                                                                                    
+*License*: Free redistribution is allowed and welcome
```

Comment:
This is too vague as a license. We require the use of a well-known open data license such as the ones in https://github.com/OSGeo/PROJ-data#about-the-data-package. Other German geodetic agencies also sometimes use the "Data licence Germany - attribution - version 2.0"

The copyright_and_licenses.csv files should also be modified

Reply:
> we don't have any licence defined for it and there is nobody who is responsible for this topic

Are you connected with the team that produced the file? Who is the owner of the intellectual property with it ? If it is you (your organization), then you can freely decide of the license for the data you deliver. CC-BY-4.0 (https://creativecommons.org/licenses/by/4.0/) is a quite popular open data license that a lot of geodetic agencies use for their grids. It cover all the open data freedoms while crediting the copyright owner of the data.